includes/Modules/Search_Console.php

```diff
@@ -156,21 +156,18 @@ public function get_debug_fields() {
 	}
 
 	/**
-	 * Returns the mapping between available datapoints and their services.
+	 * Gets map of datapoint to definition data for each.
 	 *
-	 * @since 1.0.0
+	 * @since n.e.x.t
 	 *
-	 * @return array Associative array of $datapoint => $service_identifier pairs.
+	 * @return array Map of datapoints to their definitions.
 	 */
-	protected function get_datapoint_services() {
+	protected function get_datapoint_definitions() {
 		return array(
-			// GET.
-			'sites'           => 'webmasters',
-			'matched-sites'   => 'webmasters',
-			'searchanalytics' => 'webmasters',
-
-			// POST.
-			'site'            => 'webmasters',
+			'GET:matched-sites'   => array( 'service' => 'webmasters' ),
+			'GET:searchanalytics' => array( 'service' => 'webmasters' ),
+			'GET:sites'           => array( 'service' => 'webmasters' ),
```

Comment:
`GET:sites` should come after `POST:site`.

Reply:
Updated.